config/webpack.dev.js

```diff
@@ -17,22 +19,6 @@ module.exports = (env, argv) => {
             },
             mode: emulateProdServer ? "production" : "development",
             devtool: emulateProdServer ? false : "inline-source-map",
-            module: {
```

Comment:
_loader-update_: Partially revert this change, modifying `html-loader` to do the work for us (GitHub doesn't allow suggestions on deleted lines, so you'll have to fix this manually)

```js
            module: {
                rules: [
                    {
                        test: /\.html$/,
                        use: [
                            {
                                loader: "html-loader",
                                options: {
                                    // wrap the content of the html file in the layout.html file
                                    preprocessor: (content) =>
                                        htmlLayout.replace(
                                            "<%=content%>",
                                            content
                                        ),
                                },
                            },
                        ],
                    },
                ],
            },
```